analyzers/src/SonarAnalyzer.Common/Helpers/SonarAnalysisContext.cs

```diff
@@ -39,13 +40,12 @@ public class SonarAnalysisContext
     {
         public delegate bool TryGetValueDelegate<TValue>(SourceText text, SourceTextValueProvider<TValue> valueProvider, out TValue value);
 
-        private const string SonarProjectConfigFileName = "SonarProjectConfig.xml";
+        public const string SonarProjectConfigFileName = "SonarProjectConfig.xml";
```

Comment:
There is a code smell here. It should be static read-only.

Reply:
I agree that the rule is not relevant in this context and, at the same time, we should exclude it from our profile if we don't want to follow it.